index.html

```diff
@@ -243,14 +243,15 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
 
 <section>
   <h3 id="verify" data-export="" data-dfn-type="dfn">
-The web must make it possible for people to verify the information they see
+The web must make it possible for people to verify, and assess the trustworthiness of, information.
   </h3>
   <p>
-We have a responsibility to build web technologies to counter misinformation,
-allowing information sources to be traceable and facts to be checkable.
-The concept of origin and source is core to the web's security model.
-We will make sure the new web technologies we create
-do not work against this architectural principle.
+Society relies on the integrity of public information.
+We have a responsibility to build web technologies to counter misinformation, 
+and to maintain the integrity of information for public good.
+The public needs verifiable source and context information to recognize
+trustworthy web publishers and content.
+The concepts of origin and source are core to the web's security model.
```

Comment:
The [origin](https://html.spec.whatwg.org/multipage/browsers.html#origin) is certainly core to the security model, but I don't know of a technical definition of "source" or a place the security model relies on it. What's that part of this sentence talking about?

Reply:
We have an issue open on that. We'd like to address that later, if we can. (But it does need addressing, we agree!) 
https://github.com/w3ctag/ethical-web-principles/issues/97